### DISCUSSION TITLE
Location for bug: Share Invoice button shouldn't be visible when not logged in #337

### DISCUSSION THREAD

**Discussion by glizondo** [TreyWW/MyFinances#347] [General]
Hi @TreyWW  I just got started with this bug and I am not able to find the location of the button in the UI. Are there specific steps to do beforehand? Thanks in advance

**Comment by TreyWW** [comment]
Hey @glizondo,

Were you on the correct page? On `dashboard/invoices/XZY_ID/preview/` you should see a top banner of the share button

![image](https://github.com/TreyWW/MyFinances/assets/73353716/cb5b85cb-4952-4b2d-9899-ccff7398bab7)

**Reply by glizondo** [reply]
So I see the banner in there and that is with the logged-in session and it takes me to Managed Access to Invoice #1 (Expected behavior)
![image](https://github.com/TreyWW/MyFinances/assets/80534057/f1f3417c-1851-43a2-8849-135e67160f70)

But then when not logged in I am not able to see the banner, only this:
![image](https://github.com/TreyWW/MyFinances/assets/80534057/4a97cc50-b414-41e7-8fe5-2c6510b0714c)

Sorry for the back and forth, I want to make sure I get it right :)

**Reply by TreyWW** [reply]
Ah yeah, to see the real invoice you need to go to the overview page and then press "Share" and copy the URL (or click it) or if you go on the invoice list page click on the 3 dots and press "Manage access". 

Apologies for that

**Reply by glizondo** [reply]
Got it! Going to work on fixing it. Thanks for clarifying

**Reply by TreyWW** [reply]
Amazing, thanks

**Comment by TreyWW** [comment]
It's probably due to the banner not being shown on the Overview page. I'll make that more clear :)
